### ISSUE TITLE
5-1, Dataset和DataLoader运行报错

### ISSUE THREAD

**Issue by fancyLv** [lyhue1991/eat_pytorch_in_20_days#27]
![1623604926951](https://user-images.githubusercontent.com/19291456/121816565-f57e4100-ccae-11eb-96f3-65a9ca33c82f.jpg)

**Comment by ElderlyPeng** [comment]
我也是这个错误
dl_train = DataLoader(ds_train,batch_size = 50,shuffle = True,num_workers=3)
dataloader的num_workers参数指加载线程的数量，多线程可以提升加载速度
但是我的电脑一使用多线程就出错，按官方教程来也出错。只是学习的话可以试试去掉多线程。
![image](https://user-images.githubusercontent.com/41566277/140605479-c95cc043-4ca9-456b-80a8-5e46e04faba2.png)
